src/Form/ImportVideoItemByMpxId.php

```diff
@@ -65,31 +31,14 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    if (!$video_opts = $this->loadVideoTypeOptions()) {
-      $this->messenger()->addError($this->t('There has been an unexpected problem loading the form. Reload the page.'));
-      return [];
-    }
+    $form = parent::buildForm($form, $form_state);
 
-    $form['video_type'] = [
-      '#type' => 'select',
-      '#title' => $this->t('Video type'),
-      '#description' => $this->t('Choose the video type to import the video into.'),
-      '#options' => $video_opts,
-      '#required' => TRUE,
-    ];
     $form['mpx_id'] = [
       '#type' => 'textfield',
       '#title' => $this->t('ID'),
       '#placeholder' => $this->t('Type the ID of the mpx video you want to import.'),
       '#required' => FALSE,
     ];
-    $form['actions']['#type'] = 'actions';
-    $form['actions']['submit'] = [
-      '#type' => 'submit',
-      '#value' => $this->t('Import video'),
-      '#button_type' => 'primary',
-    ];
-
```

Comment:
I don't think any of the `buildForm` contents should be moved into an abstract class in order to save a few a lines of code. I don't see this as being duplication that needs to be removed: it's 2 different forms, that do things with 2 different sets of data, and happen to share a part of their UI.